app/assets/stylesheets/modules/_nav.styl

```diff
@@ -1,6 +1,6 @@
 nav.top-nav
   top 0
-  z-index 100
+  z-index 10000
```

Comment:
A comment from me would have helped :) As you scroll down the page, the alert banners will appear on top of the top nav. I assume this was not intentional. I could have also decreased the z-index of `.notice` but I wasn't sure if that would break anything. It seemed more likely that the top nav would always be on top of everything else.